### ISSUE TITLE
[BUG] Specific feed fails to update until triggering “Reload Articles”

### ISSUE THREAD

**Issue by LemonsParty** [FreshRSS/FreshRSS#3768]
[log.txt](https://github.com/FreshRSS/FreshRSS/files/6986924/log.txt)
**Describe the bug**
The feed at https://1900hotdog.com/feed/ stops working for me every day (showing the “Blast! This feed has encountered a problem.” message. Clicking “Reload Articles” under Maintenance causes the next refresh to be successful, and then the feed goes back to failing to update again.

**To Reproduce**
1. Add the feed https://1900hotdog.com/feed/
2. Configure FreshRSS to update every half hour.
3. View the feed the next day.

**Expected behavior**
Feed updates normally and new articles appear when posted.

**Environment information (please complete the following information):**
 - Device: Raspberry Pi 3B
 - OS: Raspbian GNU/Linux 10 (buster)
 - Browser: N/A
 - FreshRSS version: 1.19.0-dev
 - Database version: 10.0.28-MariaDB-2+b1 Raspbian testing-staging
 - PHP version: 7.3.27-1~deb10u1
 - Installation type: Not sure? I followed the general installation instructions on freshrss.org to install on a Raspberry Pi.

**Additional context**
This issue doesn’t reproduce with any of the ~100 other feeds I’m subscribed too. 

Attached log file.

Is there a way to temporarily work around this by triggering that “Reload Articles” action via Terminal so I could do that every once in a while with a cron job?

**Comment by Alkarex** [comment]
It looks like there are some random problems with that feed, which I can also reproduce sometimes from command line:

```sh
curl -v -4 https://1900hotdog.com/feed/
*   Trying 198.71.233.189:443...
* Connected to 1900hotdog.com (198.71.233.189) port 443 (#0)
* ALPN, offering h2
* ALPN, offering http/1.1
* successfully set certificate verify locations:
*  CAfile: /etc/ssl/certs/ca-certificates.crt
*  CApath: /etc/ssl/certs
* TLSv1.3 (OUT), TLS handshake, Client hello (1):
* OpenSSL SSL_connect: SSL_ERROR_SYSCALL in connection to 1900hotdog.com:443
* Closing connection 0
curl: (35) OpenSSL SSL_connect: SSL_ERROR_SYSCALL in connection to 1900hotdog.com:443
```

**Comment by Alkarex** [comment]
I did a test and while it often fails, this feed works sometimes. I do not think there is much to do on our side

**Comment by LemonsParty** [comment]
Ah, fun timing, I was just about to post an update. So in trying to figure out what was going on, I accidentally stumbled on a bizarre workaround.

I experimented with checking the feed with curl as well. Every time I ran that, the next refresh in FreshRSS would work. I set up a cron job to run a curl to that address once an hour, while my feeds refreshed every 30 minutes. With that setup, every other refresh failed consistently. Updating it so the curl command runs right before every refresh of that feed now has it updating reliably every single time.

So, hooray for my problem being worked around, I guess? But something weird is going on there. I don’t reproduce this issue when testing this feed with Feedly or with iCloud feeds in Reeder, so it seems like something is going on, but I haven’t the foggiest idea what it could be. I’m wondering if the intermittent successes you had refreshing were correlated with you testing around accessing the feed via curl too? So strange.
